src/services/hyperlink-utility.js

```diff
@@ -0,0 +1,21 @@
+function supportsHyperlink() {
+  const { env } = process;
+  const supports = require('supports-hyperlinks');
+  if (supports.stdout) {
+    return true;
+  }
+  // support for Windows terminal
+  if ('WT_SESSION' in env) {
```

Comment:
How you figured out for windows we need to check for `WT_SESSION` flag? As I saw there is no exception mentioned [here](https://github.com/jamestalmage/supports-hyperlinks)

Reply:
I got the idea from this [PR](https://github.com/jamestalmage/supports-hyperlinks/pull/8/files) on the repo. Also, to verify I scrolled through the microsoft/terminal repo and found [this](https://github.com/microsoft/terminal/pull/897).